docs/search/index.asciidoc

```diff
@@ -19,53 +19,53 @@ The *Search* space in {kib} comprises the following features:
+| link:https://www.elastic.co/guide/en/enterprise-search/current/crawler.html[Documentation]
+| link:https://www.elastic.co/guide/en/enterprise-search/current/changelog.html[Enterprise Search Guide]
 
-// | Playground
-// | link:https://example.com/playground/api[API reference]
-// | link:https://example.com/playground/docs[Documentation]
-// | link:https://example.com/playground/notes[Release notes]
+| Playground
+| N/A
+| <<playground,Documentation>>
+| <<release-notes, Kibana guide>>
 
-// | Search Applications
-// | link:https://example.com/search_apps/api[API reference]
-// | link:https://example.com/search_apps/docs[Documentation]
-// | link:https://example.com/search_apps/notes[Release notes]
+| Search Applications
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/search-application-apis.html[API reference]
+| link:https://www.elastic.co/guide/en/enterprise-search/current/app-search-workplace-search.html[Documentation]
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/es-release-notes.html[Elasticsearch guide]
 
-// | Behavioral Analytics
-// | link:https://example.com/behavioral_analytics/api[API reference]
-// | link:https://example.com/behavioral_analytics/docs[Documentation]
-// | link:https://example.com/behavioral_analytics/notes[Release notes]
+| Behavioral Analytics
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/behavioral-analytics-apis.html[API reference]
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/behavioral-analytics-start.html[Documentation]
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/es-release-notes.html[Elasticsearch guide]
 
-// | Inference Endpoints
-// | link:https://example.com/inference_endpoints/api[API reference]
-// | link:https://example.com/inference_endpoints/docs[Documentation]
-// | link:https://example.com/inference_endpoints/notes[Release notes]
+| Inference Endpoints
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html[API reference]
+| <<inference-endpoints,Documentation>>
+| link:https://www.elastic.co/guide/en/elasticsearch/reference/current/es-release-notes.html[Elasticsearch guide]
 
-// | Console
-// | link:https://example.com/console/api[API reference]
-// | link:https://example.com/console/docs[Documentation]
-// | link:https://example.com/console/notes[Release notes]
+| Console
+| N/A
+| <<console-kibana,Documentation>>
+| <<release-notes, Kibana guide>>
 
-// | Search UI
-// | link:https://www.elastic.co/docs/current/search-ui/api/architecture[API reference]
-// | link:https://www.elastic.co/docs/current/search-ui/overview[Documentation]
-// | link:https://example.com/search_ui/notes[Release notes]
+| Search UI
+| link:https://www.elastic.co/docs/current/search-ui/api/architecture[API reference]
+| link:https://www.elastic.co/docs/current/search-ui[Documentation]
+| link:https://www.elastic.co/docs/current/search-ui[Search UI]
```

Comment:
```suggestion
| https://www.elastic.co/docs/current/search-ui/api/architecture[API reference]
| https://www.elastic.co/docs/current/search-ui[Documentation]
| https://www.elastic.co/docs/current/search-ui[Search UI]
```

nit:unnecessary

Reply:
I don't think Search UI has an attribute plus those docs aren't versions, so full URL is fine here